docs/changelog/0015-separate-celery-datadog-http-check.md

```diff
@@ -0,0 +1,28 @@
+# 15. Separate celery datadog http check
+
+**Date:** 2019-02-22
+
+**Optional per env:** _only required on some environments_
+
+
+## CommCare Version Dependency
+CommCare versions beyond this commit require this change to function correctly:
+[81325fbd](https://github.com/dimagi/commcare-hq/commit/81325fbd9e3131c710179f7246dbe9caccb45154)
```

Comment:
So @kaapstorm for example this is what `max_commcare_version` means for this change.